SSA/Projects/LLVMRiscV/Pipeline/const.lean

```diff
@@ -0,0 +1,2233 @@
+-- AUTOGENERATED Lean file: Constant lowering for RISC-V li instructions
+
+import SSA.Projects.LLVMRiscV.PeepholeRefine
+import SSA.Projects.LLVMRiscV.simpproc
+import SSA.Projects.RISCV64.Tactic.SimpRiscV
+
+ open LLVMRiscV
+
+    def li_riscv_n50 := [LV| {
+    ^entry ():
+      %0 = "li"() {imm = -50 : !i64}  : (!i64) -> (!i64)
+      %1 = "builtin.unrealized_conversion_cast"(%0) : (!i64) -> (i64)
+      llvm.return %1 : i64
+  }]
+def const_llvm_n50 : Com  LLVMPlusRiscV [] .pure (.llvm (.bitvec 64))  := [LV| {
```

Comment:
Your whitespace looks off here, can you try to fix your script to format things a bit nicer?

In particular, there shouldn't be spaces before that `open`, nor before that `def`, and there should an empty line after the definition (i.e., between lines 14 and 15)